src/bin/poudriere-jail.8

```diff
@@ -89,155 +85,222 @@ Except for
-.It Sy url=PATH
-Fetch from given
-.Sy PATH .
+files as well.
+.It Cm url= Ns Ar url
+Fetch distribution tarballs
+.Pq like Pa base.txz
+from the given
+.Ar url .
 Any URL supported by
 .Xr fetch 1
 can be used.
 For example:
-.Dl "url=file:///mirror/10.0"
+.Dq Cm url=file:///mirror/10.0 .
 .El
 .Pp
 Build from source options:
-.Bl -tag -width "ftp-archiveXX"
-.It Sy git Sy git+http Sy git+https Sy git+ssh
-Will use git, the -v flag to set the branch name and the
-.Sy GIT_BASEURL
+.Bl -tag -width "git[+protocol]"
+.It Cm git Ns Op Cm + Ns Ar protocol
+Use Git to download the sources.
+.Pp
+Use the
+.Fl v
+flag to set the branch name.
+The Git server address is derived from the
+.Va GIT_BASEURL
 variable in
 .Pa poudriere.conf .
-.It Sy src=PATH
-With the
+.Pp
+The following protocols are supported:
+.Cm git No (default) , Cm http , https , No and Cm ssh .
+.It Cm src= Ns Ar path
+Copy the source tree from
+.Ar path
+into the jail,
+and build it.
+This option is meant to be used with the
 .Fl b
-flag, the src tree will be copied into the jail and built.
-.It Sy svn Sy svn+file Sy svn+http Sy svn+https
+flag.
+.It Cm svn Ns Op Cm + Ns Ar protocol
 Will use SVN and the
-.Sy SVN_HOST
+.Ev SVN_HOST
+variable in
+.Pa poudriere.conf .
+Use SVN to download the sources.
+.Pp
+The SVN host address is derived from the
+.Va SVN_HOST
 variable in
 .Pa poudriere.conf .
+.Pp
+The following protocols are supported:
+.Cm svn No (default) , Cm file , http , No and Cm https .
 .El
-.It Fl f Ar filesystem
-Specifies the
-.Ar filesystem
-name (${ZPOOL}/jails/filesystem).
-.It Fl K Ar kernelname
-Install the jail with a kernel.
-If the
-.Ar kernelname
-is an empty string GENERIC will be used.
-If installing from ftp then the default kernel will be installed what ever the
-.Ar kernelname
-value is.
-.It Fl M Ar mountpoint
-Gives an alternative
-.Ar mountpoint
-when creating jail.
-.It Fl p Ar name
-This specifies which port tree to start/stop the jail with.
+.It Fl n
+When combined with
+.Fl l ,
+only display jail name.
 .It Fl P Ar patch
 Apply the specified
 .Ar patch
 to the source tree before building the jail.
+.It Fl p Ar name
+Specify the ports tree to start/stop the jail with.
+.It Fl q
+Remove the header when
+.Fl l
+is the specified mandatory option.
+Otherwise, it has no effect.
 .It Fl S Ar srcpath
 Use the specified
 .Ar srcpath
 as the
 .Fx
 source tree mounted inside the jail
 or from the host for
-.Fl m Ar null .
+.Fl m Cm null .
 .It Fl t Ar version
-instead of upgrading to the latest security fix of the jail version, you can
-jump to the new specified
-.Ar version .
-.It Fl z Ar set
-This specifies which SET to start/stop the jail with.
+Upgrade the jail to the specified
+.Ar version
+instead of upgrading to the latest security fix.
+.It Fl v Ar version
+Specify the
+.Ar version
+of
+.Fx
+to use in the jail.
+If you are using
+.Fl m Cm ftp
+then the
+.Ar version
+should in the form of
+.Dq Cm 9.0-RELEASE .
```

Comment:
Can you use a currently supported FreeBSD version while you're making changes here?